contracts/voting-verifier/src/contract.rs

```diff
@@ -64,6 +66,20 @@ pub fn execute(
     }
 }
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn query(_deps: Deps, _env: Env, _msg: QueryMsg) -> StdResult<Binary> {
-    todo!()
+pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
+    match msg {
+        QueryMsg::IsVerified { messages } => {
+            let messages = messages
+                .into_iter()
+                .map(state::Message::try_from)
+                .collect::<Result<Vec<state::Message>, connection_router::ContractError>>()
+                .map_err(|e| StdError::generic_err(e.to_string()))?;
```

Comment:
If we add:
```
impl From<ContractError> for StdError {
    fn from(value: ContractError) -> Self {
        Self::generic_err(value.to_string())
    }
}
```
to connection_router/src/error.rs, we can get rid of this `map_err`